docs/tutorials/_index.md

```diff
@@ -3,4 +3,62 @@ title: "Tutorials"
 linkTitle: "Tutorials"
 weight: 65
 type: docs
+no_list: true
 ---
+
+{{< alert title="Tip" color="tip" >}}
+While tutorials may list specific hardware, the hardware only affects the configuration of a project.
+Once you have configured your components, you can use the same code of the tutorials with any hardare.
```

Comment:
```suggestion
Once you have configured your components, you can use the same code of the tutorials with any hardware.
```

Reply:
would also suggest saying "SDK code" 
or maybe instead "configured your components with Raw JSON or the app Config Builder"
not necessary but could help differentiate what code means here as ppl generally probably think configuration is code (even if json is not quite code)

Reply:
Going to stick with what we have and link to the configure page once its up. I don't think anyone would call it SDK code. But I'm open to other suggestions.